conda_smithy/cli.py

```diff
@@ -52,6 +53,23 @@ def generate_feedstock_content(target_directory, source_recipe_dir):
         with feedstock_io.write_file(forge_yml) as fh:
             fh.write("{}")
 
+    # merge in the existing configuration in the source recipe directory
+    forge_yml_recipe = os.path.join(source_recipe_dir, "conda-forge.yml")
+    yaml = YAML()
+    if os.path.exists(forge_yml_recipe):
+        os.unlink(os.path.join(target_recipe_dir, "conda-forge.yml"))
+        try:
+            with open(forge_yml_recipe, "r") as fp:
+                _cfg = yaml.load(fp.read())
+        except:
+            _cfg = {}
+
+        with open(forge_yml, "r") as fp:
+            _cfg_feedstock = yaml.load(fp.read())
+            merge_dict(_cfg, _cfg_feedstock)
+        with open(forge_yml, "w") as fp:
+            yaml.dump(_cfg_feedstock, fp)
```

Comment:
I think you need to git add the file afterwards. You can do `yaml.dumps` and use `feedstock_io.write_file`

Reply:
I think the commit on https://github.com/conda-forge/conda-smithy/blob/589689ed9af2cf1fbaaf4e9ae39f0b042e291382/conda_smithy/cli.py#L139 takes care of that

Reply:
I mean the `git add`, not `git commit`.

Reply:
we don't actually explicitly call git add anywhere in that phase

Reply:
`feedstock_io.write_file` does the `git add`

Reply:
see here: https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/feedstock_io.py#L53

Reply:
Ah gotcha, updated appropriately